### ISSUE TITLE
bug(autocomplete): Value not consistently announced upon selection

### ISSUE THREAD

**Issue by jelbourn** [angular/components#22990]
When selecting an option from mat-autocomplete with the enter key, nothing announces the selected item. I confirmed this behavior with ChromeVox: [video](https://drive.google.com/file/d/11uM-AOQ2TAGyWOWqvDt8fQfZ3oAyPnK1/view)

I wasn't able to get a consistent reproduction with VoiceOver, but ChromeVox is consistent. Ideally I'd like to avoid using `LiveAnnouncer` for this- I feel like there should be a way to set the value of the text input such that the control announces the value. This might also end up being a ChromeVox bug, but we should check NVDA and JAWS as well. 

Tangentially related to #21684

**Comment by zarend** [comment]
closing because I believe this has been fixed by https://github.com/angular/components/pull/21870

**Comment by zarend** [comment]
Reopening because I closed this by mistake 🤦 . I meant to close issue no. 13952 instead.

**Comment by zarend** [comment]
Closing this as "Expected Behavior" because this only happens on ChromeVox, and the [w3c Example 1: List Autocomplete with Manual Selection](https://www.w3.org/TR/wai-aria-practices-1.1/examples/combobox/aria1.1pattern/listbox-combo.html) has the same behavior. JAWS and VoiceOver announce on selection while ChromeVox does not.
